lte/gateway/python/magma/pipelined/openflow/flows.py

```diff
@@ -654,17 +654,17 @@ def _check_resubmit_action(actions, parser):
 
 
 def send_stats_request(
-    datapath, tbl_num, cookie: hex = 0,
-    cookie_mask: hex = 0, retries: int = 3,
+    datapath, tbl_num, cookie: int = 0,
+    cookie_mask: int = 0, retries: int = 3,
 ):
     """
     Send a stats request msg 
     Args:
         datapath (ryu.controller.controller.Datapath):
             Datapath to query from
         table (int): Table number to query for
-        cookie (hex): cookie value for the request
-        cookie_mask (hex): cookie mask for the request
+        cookie (int): a hex cookie value for the request
+        cookie_mask (int): a hex cookie mask for the request
```

Comment:
What does "hex" mean in this context?

Reply:
All right, so it is just an int for Python. I guess they write it in hexadecimal because the variable is used as a bitmask.